pkg/operators/wasm/wasm_test.go

```diff
@@ -613,3 +614,48 @@ func TestSyscall(t *testing.T) {
 	err = runtime.RunGadget(gadgetCtx, nil, params)
 	require.NoError(t, err, "running gadget")
 }
+
+func TestPerf(t *testing.T) {
+	utilstest.RequireRoot(t)
+
+	t.Parallel()
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*2)
+	t.Cleanup(cancel)
+
+	ociStore, err := orasoci.NewFromTar(ctx, "testdata/perf.tar")
+	require.NoError(t, err, "creating oci store")
+
+	gadgetCtx := gadgetcontext.New(
+		ctx,
+		"perf:latest",
+		gadgetcontext.WithDataOperators(ocihandler.OciHandler),
+		gadgetcontext.WithOrasReadonlyTarget(ociStore),
+	)
+
+	runtime := local.New()
+	err = runtime.Init(nil)
+	require.NoError(t, err, "runtime init")
+	t.Cleanup(func() { runtime.Close() })
+
+	stop := make(chan bool)
+	go func() {
+		for {
+			select {
+				case <-stop:
+					return
+				default:
+					fd, err := syscall.Creat("/tmp/test-perf-array", 0)
```

Comment:
Is this for generating an event that triggers the ebpf program to submit info on the perf event array? If that's the case, a comment will help to clarify it.  
Also, should it use a timer to avoid creating a lot of events?